std_msgs/Header header
int32 erpm
int32 rpm
float32 duty_cycle
float32 current